management/grant_test.go

```diff
@@ -9,7 +9,7 @@ func TestGrant(t *testing.T) {
 	t.Run("List", func(t *testing.T) {
 		var gs []*Grant
 		gs, err = m.Grant.List()
-		if err != nil {
+		if err == nil {
```

Comment:
This test now verifies that when we call `List()` an error should be expected. Are you sure this is right?

Reply:
You're right with List, then the old situation is true. Reverted.